packages/keyring-controller/src/KeyringController.ts

```diff
@@ -841,7 +872,19 @@ export class KeyringController extends BaseControllerV2<
   // QR Hardware related methods
 
   /**
-   * Get qr hardware keyring.
+   * Get QR Hardware keyring.
+   *
+   * @returns The QR Keyring if defined, otherwise undefined
+   */
+  getQRKeyring(): QRKeyring | undefined {
+    // QRKeyring is not yet compatible with Keyring type from @metamask/utils
+    return this.#keyring.getKeyringsByType(
```

Comment:
Nit: We could update `getOrAddQRKeyring` to use this method as well, to make that method a bit more concise